Makefile

```diff
@@ -20,11 +20,8 @@ define DD_SDK_BASE_XCCONFIG
 // - DD_SDK_COMPILED_FOR_TESTING - conditions the SDK code compiled for testing\n
 SWIFT_ACTIVE_COMPILATION_CONDITIONS = $(inherited) DD_SDK_ENABLE_INTERNAL_MONITORING DD_SDK_COMPILED_FOR_TESTING\n
 \n
-// To build only active architecture for all configurations.\n
-// TODO: RUMM-1200 We can perhaps remove this fix when carthage supports pre-build xcframeworks.\n
-//		 The only "problematic" dependency is `CrashReporter.xcframework` which doesn't produce\n
-//		 the `arm64-simulator` architecture when compiled from source. Its pre-build `CrashReporter.xcframework`\n
-//		 available since `1.8.0` contains the `ios-arm64_i386_x86_64-simulator` slice and should link fine in all configurations.\n
+// To build only active architecture for all configurations. This gives us ~10% build time gain\n
+// in targets which do not use 'Debug' configuration.\n
 ONLY_ACTIVE_ARCH = YES\n
```

Comment:
I did this measurement in `DatadogIntegrationTests` (it doesn't use `Debug` configuration):
* with `ONLY_ACTIVE_ARCH = YES`: 65s, 65s, 75s
* without `ONLY_ACTIVE_ARCH = YES`: 72s, 81s, 77s

It is justified to have higher times when building all architectures, as produced artifacts contain more slices, e.g.:
```
❯ lipo -info /Users/.../Release-iphonesimulator/Datadog.framework/Datadog
Architectures in the fat file: /.../Release-iphonesimulator/Datadog.framework/Datadog are: x86_64 arm64 
```
vs:
```
❯ lipo -info /Users/.../Release-iphonesimulator/Datadog.framework/Datadog
Architectures in the fat file: /.../Release-iphonesimulator/Datadog.framework/Datadog are: x86_64
```